html/inc/email.inc

```diff
@@ -63,11 +64,18 @@ function send_email($user, $subject, $body, $body_html=null, $email_addr=null) {
         } else if (defined('EMAIL_FROM')) {
             $headers = "From: ". EMAIL_FROM;
         }
-        if ($email_addr) {
-            return mail($email_addr, $subject, $body, $headers);
-        } else {
-            return mail($user->email_addr, $subject, $body, $headers);
+        if (!$email_addr) {
+            $email_addr = $user->email_addr;
+        }
+        if ($body_html) {
+            $body = "<html>\n"; 
+            $body .= '<body style="font-family:Verdana, Verdana, Geneva, sans-serif; font-size:12px;">\n'; 
+            $body = $body_html; 
+            $body .= "</body>\n"; 
+            $body .= "</html>\n"; 
+            $headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n"; 
```

Comment:
Given the charset iso-8859-1, if a user has registered a username with Chinese characters, how will this be rendered on the email client?